src/components/Widget/index.tsx

```diff
@@ -169,6 +172,10 @@ export default function Widget({
         locale={locale}
         theme={theme}
         width={width}
+        dialog={dialog}
+        dialogOptions={{
+          animationType: dialog ? DialogAnimationType.FADE : DialogAnimationType.SLIDE,
+        }}
```

Comment:
I'm seeing some animation issues here w/ the corners

https://user-images.githubusercontent.com/39385577/219112929-21c909e2-feff-4afc-a3b3-0d058f7a6119.mov